### DISCUSSION TITLE
How to catch the raised HTTPException with this logging module(fastapi_log)?

### DISCUSSION THREAD

**Discussion by VSangarya** [fastapi/fastapi#8352] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, HTTPException
from pydantic import BaseModel
from fastapi_log.log_request import LoggingRoute

app = FastAPI()
app.router.route_class = LoggingRoute

uni_val = ""

class Auth(BaseModel):
    name : str
    password : str

@app.post("/get_val")
def get_val_func(data : Auth):
    global uni_val
    # A function which generates a value between 1 and 0 and stores it in global variable "uni_val"
    # Code

@app.get("ret_val")
def ret_val_func():
    global uni_val
    if uni_val > 0 :
        return uni_val
    else:
        raise HTTPException(
            status_code = 401,
            detail = { "code" : 401,
                       "message" : "Unauthorized" }
            )
```

### Description

Hi,

I'm use the fastapi_log library(https://pypi.org/project/fastapi-log/) (https://github.com/12345k/fastapi-log) in my FastAPI project.

It works great to log request and responses(return statements), but it does not log FastAPI HTTPExceptions(which I used in some parts of the program instead of return statements).

How can I edit the source code for log_request.py (https://github.com/12345k/fastapi-log/blob/main/fastapi_log/log_request.py) to be able to log responses from HTTPExceptions as well ?

This isn't an issue with FastAPI, but I wanted any suggestions or possible solutions to log using the fastapi_log module, the code and message in the HTTPException raised.

Thank you

### Operating System

Linux

### FastAPI Version

0.75.2

### Python Version

3.8.10

**Comment by panla** [comment]
```python
from fastapi import FastAPI, Request
from fastapi.exceptions import HTTPException

def register_exception(app: FastAPI):

    @app.exception_handler(HTTPException)
    async def http_exception_handler(request: Request, exc: HTTPException):

        # your logic
        pass

app = FastAPI()

register_exception(app)

```
